<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookForm</class>
 <widget class="QDialog" name="BookForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formularz</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="BookFormLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="TitleLabel">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Tytu≈Ç</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="TitleLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="AuthorLabel">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Autor</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="AuthorLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="PublishDateLabel">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Rok wydania</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="PublishDateLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="CountryPublishLabel">
      <property name="text">
       <string>Kraj wydania</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="CountryPublishLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="IsbnNumberLabel">
      <property name="text">
       <string>Numer ISBN</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="IsbnNumberLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="ButtonLayout">
    <item>
     <widget class="QPushButton" name="SaveBtn">
      <property name="text">
       <string>Zapisz</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CancelBtn">
      <property name="text">
       <string>Anuluj</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
